content/news/2021-08-10-bevys-first-birthday/index.md

```diff
@@ -0,0 +1,257 @@
+
+Our community is also extremely welcoming: we have an active [Github Q&A platform](https://github.com/bevyengine/bevy/discussions/categories/q-a) and [#help channel](https://discord.com/invite/bevy) on Discord (with over 82,000 messages). If you haven't already, [hop in to our Discord](https://discord.com/invite/bevy) and say hi!
+
+### Development Pace
+
+Just look at all of the releases above and the sheer size of their feature sets! The upcoming Bevy 0.6 is gearing up to be another big one and I doubt we'll slow down any time soon. The results speak for themselves so I'll just shut up now.
+
+### Experimentation and Iteration
+
+We didn't just build things once and assume that was the best we could do. When we started [I called out](/news/scaling-bevy/#what-made-bevy-good) that we needed the freedom to experiment and iterate. I put a big "stability warning" wherever I could to help manage expectations. And we sure did iterate! We [rewrote the transform system](https://bevyengine.org/news/bevy-0-2/#transform-system-rewrite) ... [twice](https://bevyengine.org/news/bevy-0-3/#transform-re-rewrite). We [completely rewrote Bevy ECS from scratch](https://bevyengine.org/news/bevy-0-5/#bevy-ecs-v2), leaving our hecs ECS roots behind (but not forgotten). We have a [new brand new renderer](https://github.com/bevyengine/bevy/issues/2535) in the works. I've lost count of the number of "next generation Bevy UI" prototypes and designs people have put together.
+
+I know the lack of stability has been tough for some people, but I think this is the only way to build The Best Game Engine™ in a collaborative way. Things will start to stabilize soon ... and the wait will be worth it I promise.
+
+### The Bevy App Model
+
+This year we invested heavily in what I call The Bevy App Model. Bevy {{rust_type(type="struct", crate="bevy_app", name="App", no_mod=true, plural=true)}} are easy to understand, ergonomic to write, and modular via {{rust_type(type="trait", crate="bevy_app", name="Plugin", no_mod=true, plural=true)}}. My goal was to blur the lines between engine developers and app developers. I think we absolutely nailed it:
+
+1. There is no "scripting interface" separating "engine logic" from "app logic". We use a single language (Rust) for the whole stack. Rust feels modern with "high level" niceties while retaining low level performance and control. In my opinion, Bevy Apps are often simpler and more expressive than high level equivalents like Unity or Godot, thanks to the state-of-the-art [Bevy ECS](https://github.com/bevyengine/bevy/tree/main/crates/bevy_ecs). And under the hood Bevy Apps _are_ simpler because there are no internal translation layers between languages like C++ and scripting languages like C#:
+    ```rust
+    use bevy::prelude::*;
+
+    // This is a complete, self-contained, cross platform Bevy App
+    fn main() {
```

Comment:
```suggestion
    #[bevy_main]
    fn main() {
```
Otherwise this is not portable to android and ios.

Reply:
Definitely true, but this was an intentional omission. Extra lines of code distract from the point being made. The app _is_ still  cross platform, as it could run on Windows, Mac, Linux, and Web. It could be _more_ cross platform, but I chose the option that helped drive the point home / looked more like normal rust code.